spec/requests/api/v1/upload_request_spec.rb

```diff
@@ -63,7 +63,7 @@
     expect(json).to have_key(:data)
     expect(json[:data]).to be_a(Hash)
     expect(json[:data]).to have_key(:id)
-    expect(json[:data][:id]).to be_a(String)
+    expect(json[:data][:id].to_s).to be_a(String)
```

Comment:
I think if this id comes back as `nil` then we actually _do_ want this test to fail?

Reply:
I thought that may be the case, I wasn't sure and didn't want travis to fail on my build since I didn't touch that functionality. I'll push it up without that and see what Travis says

Reply:
Huh, it failed - that's weird 🤔

Reply:
Yeah, I'm not sure why it was returning nil but that's why I changed it to have the to_s. It failed once I pulled down before I added anything so I'm not quite sure what is going on there.